users/users.go

```diff
@@ -9,8 +9,14 @@ package users
 
 import (
 	"context"
+	"regexp"
+	"strings"
+)
 
-	"github.com/asaskevich/govalidator"
+var (
+	userRegexp    = regexp.MustCompile("^[a-zA-Z0-9!#$%&'*+/=?^_`{|}~.-]+$")
```

Comment:
The email user (the local part) and the host (the domain part) can also have non ASCII UTF-8 characters. These  regex expressions will make them invalid.

Do we want to enable email addresses which have UTF-8 characters as well?